README.md

```diff
@@ -30,7 +38,9 @@ This SDK supports the following platforms:
 
 **WebWorkers:** The browser bundle supports running in a [Web Worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API) context. You can also use the [modular variant](#modular-tree-shakable-variant) of the library in Web Workers.
 
-We regression-test the library against a selection of those (which will change over time, but usually consists of the versions that are supported upstream, plus old versions of IE).
+We test the library against a selection of browsers using their latest versions. Please refer to [the test-browser GitHub workflow](./.github/workflows/test-browser.yml) for the set of browsers that currently undergo CI testing.
+
+We regression-test the library against a selection of Node.js version, which will change over time. We will always support and test against current LTS Node.js versions, and optionally some older versions that are supported upstream. We reserve the right to drop support for non-LTS versions in a non-major release. We will update the `engines` field in [package.json](./package.json) whenever we change the Node.js versions supported by the project. Please refer to [the test-node GitHub workflow](./.github/workflows/test-node.yml) for the set of versions that currently undergo CI testing.
```

Comment:
- "selection of Node.js <del>version</del> versions"
- "optionally some older versions that are _supported upstream_" — I know this language was already there, but I don't really know what it means; do you?

Reply:
> "optionally some older versions that are supported upstream" — I know this language was already there, but I don't really know what it means; do you?

Yes, "upstream" refers to our dependencies we have in ably-js, in this case the ones we use in node.js bundle (`@ably/msgpack-js`, `got` and `ws` atm). Then, the entire sentence means that if those libraries still support older versions of node.js, we may decide to support them too (hence "optionally"). Conversely, if all our dependencies require node.js 18+, there is no way for us to support an older node.js 16, even if we wanted to.

I agree it's a bit confusing the way it's written right now, I also had to think about what it meant for a while..
I think it's a bit more cleared this way:
```
and optionally some older versions that are still supported by upstream dependencies
```

Will change to that.